plugins/tiddlywiki/markdown/files/remarkable-katex.js

```diff
@@ -0,0 +1,20 @@
+"use strict";const rkatex=(md,options)=>{const backslash='\\';const dollar='$';const opts=options||{};const delimiter=opts.delimiter||dollar;if(delimiter.length!==1){throw new Error('invalid delimiter');}
```

Comment:
I think it would be good, to add a link to the license here (above line 1 ), so it will be included into the plugin and the github repo will be referenced, so we can easily find it. 

/*
License see: https://github.com/bradhowes/remarkable-katex/blob/master/LICENSE
*/

You can leave the license text from above in the repo. So everything will be fine.

Reply:
Or you could just link to the repo..

Reply:
Ideally we do not modify source files bought in from external projects. Instead, we hack them via the tiddlywiki.files file, as below.

With other plugins that incorporate 3rd party libraries (eg JSZip) we do tend to expose the underlying license as a tiddler.

Reply:
Ahh ok -- one thing I did was to minify the JS from remarkable-katex. Should I include the un-minified JS instead?

Reply:
In that situation I would suggest including the unminified JS  in the files directory, but not pulling it in via `tiddlywiki.files`

Reply:
SG -- added the original and renamed the minified version.